.github/workflows/reusable-setup.yml

```
@@ -0,0 +1,220 @@
+        uses: dtolnay/rust-toolchain@stable
+        with:
+          toolchain: ${{ inputs.rust-toolchain }}
+          components: ${{ inputs.rust-components }}
+          targets: ${{ inputs.rust-targets }}
+
+      # SYSTEM DEPENDENCIES (standardized list)
+      - name: Install system dependencies
+        if: inputs.install-system-deps
+        run: |
+          echo "Installing standard system dependencies..."
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)